### ISSUE TITLE
NDPPP error in selfcal-applyFR (step #35)

### ISSUE THREAD

**Issue by mknapp55** [lofar-astron/PiLL#37]
Similar to #30  

```
LOFAR Exception detected: [Exception: No parameter value found for RotationMeasure:CS001LBA]
in function void LOFAR::DPPP::ApplyCal::updateParms(double)
(/build/lofar-b_IP79/lofar-2.20.2/CEP/DP3/DPPP/src/ApplyCal.cc:381)
```

**Comment by mknapp55** [comment]
Upon reflection, this may be due to the modified `CORRECTED_DATA` column ( #38 ).  Has anyone else seen this?

**Comment by adrabent** [comment]
This is maybe a stupid question, but do you have removed `CS001LBA`?

**Comment by mknapp55** [comment]
No, at least not deliberately.  I suppose it's possible that it could be flagged out somehow, but I haven't seen this problem in previous pipeline runs on the same data.  `CS001LBA` is my default reference station, though I could try using a different one.

**Comment by adrabent** [comment]
Please check in your `job_directory` all the parmdb tables `pipeline.globaldbFR_loop?/sol000_instrument-?/NAMES` (according to your loop number and the frequency band) whether there is the entry `RotationMeasure:CS001LBA` in the `NAME` column
